lib/headerDeps.js

```diff
@@ -0,0 +1,8 @@
+const config = require('../lib/config')
+const util = require('../lib/util')
+
+const headerDeps = (buildConfig = config.defaultBuildConfig, options) => {
+  util.headerDeps()
```

Comment:
there's no reason to wrap this in a method. This isn't a general purpose (util) method so it should just directly call https://github.com/brave/brave-browser/pull/10599/files#diff-3deb3f32958bb937ae05c6f3e4abbdf5R181